front/test/Content-test.js

```diff
@@ -4,17 +4,29 @@ import { shallow, mount } from 'enzyme'
 import sinon from 'sinon'
 import Content from '../src/Content'
 import Site from '../src/Site'
+import Charts from '../src/Charts'
 import SearchBar from '../src/SearchBar'
 
 const arraySite = require('./resources/array-sites')
 
 describe("Content", () => {
 
+  it('should have a content class', () => {
+    const wrapper = shallow(<Content sites={arraySite} />)
+
+    expect(wrapper.hasClass('content')).to.equal(true)
+  })
 
   it('should have a site-table class', () => {
     const wrapper = shallow(<Content sites={arraySite} />)
 
-    expect(wrapper.hasClass('site-table')).to.equal(true)
+    expect(wrapper.find('.content').find('.site-table')).to.have.length(1);
```

Comment:
Pour changer `expect(wrapper.hasClass('site-table')).to.equal(true)` pour `expect(wrapper.find('.content').find('.site-table')).to.have.length(1)`

On perd en lisibilité